.github/workflows/release.yml

```diff
@@ -754,3 +754,35 @@ jobs:
       - name: Push to DockerHub
         run: |
           docker push ${{ steps.docker_env.outputs.docker_image }}
+
+  generate_sbom:
+    runs-on: ubuntu-latest
+    needs: create_release
+    steps:
+      - name: checkout code
+        uses: actions/checkout@v3 
+
+      - name: Grant the safe directory for git
+        run: |
+          git config --global --add safe.directory $(pwd)
+      - name: generate sbom
+        uses: advanced-security/sbom-generator-action@v0.0.1
+        id: sbom
+        env:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+
+      - name: install gh
```

Comment:
If we will install `gh`, why don't we use [gh-sbom](https://github.com/advanced-security/gh-sbom) instead? This enables an extension of the `gh` tool and generates the SBOM with one command.

Reply:
For this , i thought of being consistent with other jobs in the release.yml file , which had gh installed this way, the only purpose of installing gh was to upload the release tar.gz package. if we choose to move forward with uploading it as an artifact we don't need gh.

Reply:
You still need gh to upload the assets to the release page. Just want to make sure that the reason you choose the action workflow instead of the gh extension.

Reply:
ohkay, so someone in cncf slack recommended the above sbom workflow action, hence i went forward with this.